db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.1].define(version: 2024_03_15_132443) do
+ActiveRecord::Schema[7.2].define(version: 2024_03_15_132443) do
```

Comment:
Probably a silly question but how did this change? 😅

Reply:
When you run migrations, it sets the ActiveRecord schema version to that of current ActiveRecord, even if no migration happens. The gem will have been updated but the migrations not run. I've run the migrations.